presto-hive/src/main/java/com/facebook/presto/hive/ParquetHiveRecordCursor.java

```diff
@@ -808,6 +805,26 @@ public ParquetListConverter(Type prestoType, String columnName, GroupType listTy
             }
         }
 
+        //copied over from Apache Hive
+        private boolean isElementType(parquet.schema.Type repeatedType, String parentName)
+        {
+            if (repeatedType.isPrimitive() ||
+                    (repeatedType.asGroupType().getFieldCount() != 1)) {
```

Comment:
Having a group with 0 fields is not possible, please see [this](https://github.com/apache/parquet-mr/blob/master/parquet-column/src/main/java/org/apache/parquet/schema/Types.java#L622).